### DISCUSSION TITLE
Possible encryption detected (high entropy)

### DISCUSSION THREAD

**Discussion by eniocc** [sepinf-inc/IPED#1656] [General]
Hello everybody. I'm processing a HD using IPED and I came across the following error message "1. Possible encryption detected (High entropy (7.85)) (Sector offset: 0) ...". In IPED 21k multimedia files appeared while in FTK Imager many other files appeared. Am I doing something wrong? Thanks in advance.

**Comment by wladimirleite** [comment]
Hello @eniocc!

What is the kind of partition FTK Imager detected?
What IPED version are you using?

In some cases TSK (used by IPED) fails to identify the file system, but FTK Imager does it successfully.
In such cases, as a workaround, it is possible to export the content to an AD1 and then use that AD1 as IPED's input.

**Reply by eniocc** [reply]
I'm using IPED 4.1.2. The file system detected by FTK is NTFS.
I'm already converting to AD1 and will process it at IPED. At the end, post the result here. Thanks in advance.

**Reply by wladimirleite** [reply]
NTFS should be handled properly by TSK...
You are using Windows, right?
Try running tsk_loaddb.exe directly on your image later.
Which format/extension are you using (E01, DD)?

**Reply by eniocc** [reply]
Yes, I'm using and E01. Trying tsk_loaddb.exe now.

**Reply by eniocc** [reply]
> NTFS should be handled properly by TSK... You are using Windows, right? Try running tsk_loaddb.exe directly on your image later. Which format/extension are you using (E01, DD)?

The output was: "Error: Cannot determine file system type (Sector offset: 34, Partition Type: Microsoft reserved partition). Database stored at: .."

**Reply by wladimirleite** [reply]
> The output was: "Error: Cannot determine file system type (Sector offset: 34, Partition Type: Microsoft reserved partition). Database stored at: .."

This is usually a small partition that does not store user data, so this error could be ignored.

**Reply by eniocc** [reply]
> > The output was: "Error: Cannot determine file system type (Sector offset: 34, Partition Type: Microsoft reserved partition). Database stored at: .."
> 
> This is usually a small partition that does not store user data, so this error could be ignored.

You are right, but the process has stopped for an unknown reason.
After running tsk_loaddb.exe directly on the image and running the IPED process again with the same command and parameters, the error is gone and it is processing perfectly as usual.
Thanks a lot for the support.
